### PR TITLE
Get ballot hash before auditing or casting

### PR DESCRIPTION
**Pull request by slickepinne**
With this PR, the `ballot_hash` is available after `encrypting` the vote. Therefore, it can be displayed before deciding to `audit` or `cast` a vote.

### CODE REVIEW

**Review by agustibr — APPROVED** [review]
👍

**Review by beagleknight — APPROVED** [review]
Good job 🤩 !

File: decidim-bulletin_board-app/cypress/integration/sandbox/election.page.js [hunk 1]
```diff
@@ -141,13 +141,31 @@ export class ElectionPage {
...
+  }
+
+  /**
+   * Assert that the ballot hash is present
+   *
+   * @returns {undefined}
+   */
+  assertBallotHashIsPresent() {
+    cy.findByText("Encrypt vote").should("be.disabled");
+    cy.get(".ballot-hash").should("include.text", "Your ballot identifier is:");
```
Comment (beagleknight) [inline decidim-bulletin_board-app/cypress/integration/sandbox/election.page.js:160]: Avoid using selectors if possible. I think it's better to just use `cy.findbyText("Your ballot identifier is:").should("be.visible")`.

**Review by beagleknight — APPROVED** [review]
Great!

**Review by leio10 — APPROVED** [review]
Great worked! I love the new names and the responsibilities reorganization ❤️ 😻

**Pull request merged**